en/docs/SetupAndInstall/UpgradingWSO2APIManager/upgrading-from-220-to-300.md

```diff
@@ -1,222 +1,1632 @@
+
+6.  Move all your Synapse configurations to API-M 3.0.0 pack.
     -   Move your Synapse super tenant configurations.
-        Copy the contents in the `<CURRENT_API-M_HOME>/repository/deployment/server/synapse-configs/default` directory and replace the contents in the `<API-M_2.6.0_HOME>/repository/deployment/server/synapse-configs/default` directory with the copied contents.
+        Copy the contents in the `<OLD_API-M_HOME>/repository/deployment/server/synapse-configs/default` directory and replace the contents in the `<API-M_3.0.0_HOME>/repository/deployment/server/synapse-configs/default` directory with the copied contents.
     -   Move all your tenant Synapse configurations.
-        Copy the contents in the `<CURRENT_API-M_HOME>/repository/tenants` directory and replace the contents in the `<API-M_2.6.0_HOME>/repository/tenants` directory with the copied contents.
+        Copy the contents in the `<OLD_API-M_HOME>/repository/tenants` directory and replace the contents in the `<API-M_3.0.0_HOME>/repository/tenants` directory with the copied contents.
 
-#### Step 1.2 - Optionally, migrate the configurations for WSO2 API-M Analytics
+    !!! warning
+        When moving the Synapse configurations, **do not replace** the following set of files as they contain some modificatiosn in API-M 3.0.0 version.
 
-!!! note
-This step is **only required** if you have WSO2 API-M-Analytics configured in your current deployment.
-!!! info
-As you are upgrading from WSO2 API-M Analytics 2.2.0, in order migrate the configurations required to run WSO2 API-M Analytics for WSO2 API-M 2.6.0 carryout the same instructions as mentioned in
-[Upgrading from 2.5.0 to 2.6.0 - Step 1.2 - Optionally, migrate the configurations for WSO2 API-M Analytics](https://docs.wso2.com/display/AM260/Upgrading+from+the+Previous+Release#250) .
+        -   _RevokeAPI_.xml
+        -   _cors_request_handler_.xml
+        -   main.xml
 
-### Step 2 - Upgrade API Manager to 2.6.0
+7. If you manually added any custom OSGI bundles to the `<API-M_2.6.0_HOME>/repository/components/dropins` directory, copy those to the `<API-M_3.0.0_HOME>/repository/components/dropins` directory. 
+
+8. If you manually added any JAR files to the `<API-M_2.6.0_HOME>/repository/components/lib` directory, copy those and paste them in the `<API-M_3.0.0_HOME>/repository/components/lib` directory.
+
+9. WSO2 API Manager 3.0.0 has been switched from log4j to log4j2. You will notice that there is a log4j2.properties file in the `<API-M_3.0.0_HOME>/repository/conf/` directory instead of the log4j.properties file. Follow [Upgrading to Log4j2](../UpgradingWSO2APIManager/upgrading-to-log4j2.md) to migrate your existing log4j.properties file to log4j2.properties file.
```

Comment:
Shall we use wording as upgrade instead of switch? Ex- Log4j version has been upgraded to log4j2 in API Manager 3.0.0. WDYT?

Reply:
+1